src/guides/bioinformatics/exporting-data-for-auspice.md

```diff
@@ -0,0 +1,149 @@
+## How data is exported by Augur for visualisation in Auspice
+
+This page is intended to describe which data underpins the different parts of an Auspice visualisation, and how this data can be produced via Augur. It is not a comprehensive guide to how to generate individual pieces of data nor how to interpret them in Auspice. 
+
+### Auspice components
+
+It’s helpful to start in auspice and then work backwards to augur. The following (annotated) screenshot shows the components of a typical visualisation, which is based on a dataset JSON (sometimes called an Auspice JSON). 
+
+Generally, data is defined on individual tree nodes in this dataset (via `node.node_attrs: Dictionary` or `node.branch_attrs: Dictionary` ) or within `meta: Dictionary` . We will refer to these below, however the best place to really understand the data structures is  [the JSON schema](https://github.com/nextstrain/augur/blob/master/augur/data/schema-export-v2.json).  
```

Comment:
I would also consider a different notation for types than `dotted.name: type`, as to my eye/mind it seems too much like `key: value` to me when reading. Perhaps something like

> `dotted.name` (dictionary)

or similar.

Reply:
👍 I think I've managed to change all of these